src/http/api/views/account.view.ts

```diff
@@ -342,6 +362,20 @@ export class AccountView {
         };
     }
 
+    private async getBlueskyIntegrationData(requestUserAccountId: number) {
+        const blueskyHandle = await this.db(
+            'bluesky_integration_account_handles',
+        )
+            .where('account_id', requestUserAccountId)
+            .first()
+            .then((row) => row?.handle);
+
+        return {
+            blueskyEnabled: !!blueskyHandle,
+            blueskyHandle: blueskyHandle || null,
+        };
```

Comment:
Should we return only a single field here, `blueskyHandle`?

If the value is null, the client renders Bluesky sharing off.
If the value exists, the client renders Bluesky sharing on and displays the handle.